crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -96,11 +94,8 @@ pub struct AuthorityPerpetualTables {
     // and never changed
     pub(crate) root_state_hash_by_epoch: DBMap<EpochId, (CheckpointSequenceNumber, Accumulator)>,
 
-    /// A singleton table that stores the current epoch number. This is used only for the purpose of
-    /// crash recovery so that when we restart we know which epoch we are at. This is needed because
-    /// there will be moments where the on-chain epoch doesn't match with the per-epoch table epoch.
-    /// This number should match the epoch of the per-epoch table in the authority store.
-    current_epoch: DBMap<u64, u64>,
+    /// Parameters of the system fixed at the epoch start
+    pub(crate) epoch_start_configuration: DBMap<EpochId, EpochStartConfiguration>,
```

Comment:
Why is this a map from EpochId? I think we just need it for the current epoch

Reply:
fair point, made the change